work-item-link-category-blackbox_test.go

```diff
@@ -292,7 +292,7 @@ func (s *workItemLinkCategorySuite) TestShowWorkItemLinkCategoryOK() {
 	_, linkCat2 := test.ShowWorkItemLinkCategoryOK(s.T(), nil, nil, s.linkCatCtrl, *linkCat.Data.ID)
 
 	require.NotNil(s.T(), linkCat2)
-	require.NotNil(s.T(), linkCat2.Links, "The link category should include a self link")
+	require.NotNil(s.T(), linkCat2.Data.Links, "The link category should include a self link")
```

Comment:
No, this is correct. `Links` is optional (a pointer) but `Self` is a required member.